docs/jsdoc/template-functions.jsdoc

```diff
@@ -49,11 +59,12 @@
  */
 
 /**
- * Escapes a string using HTML/XML escaping rules.
+ * The callback called by ClientFunction to include files at runtime with `include()`
  *
- * @callback EscapeCallback
- * @param  {String} markup Input string
- * @return {String}        Escaped string
+ * @callback IncludeCallback
+ * @param  {String} path      path to be included
+ * @param  {Object} [data]    data passed to the template
```

Comment:
What's inconsistent? :confused:  I can't see anything out of line.

Probably I'm just missing something.